invenio_app_ils/ui/src/invenio_app_ils/authentication/services/SessionManager.js

```diff
@@ -0,0 +1,76 @@
+import { authenticationService } from './AuthenticationService';
+
+class SessionManager {
+  constructor() {
+    let tokenData;
+
+    if (process.env.NODE_ENV === 'production') {
+      let encodedToken = authenticationService.getTokenFromDOM();
+      if (encodedToken) {
+        tokenData = authenticationService.decodeProductionToken(encodedToken);
+        this.setUser(encodedToken, tokenData);
+      } else {
+        this.setAnonymous();
+      }
+    } else {
+      if (process.env.REACT_APP_JWT_TOKEN) {
+        tokenData = {
+          sub: process.env.REACT_APP_USER_ID,
```

Comment:
There are newly introduced environment variables, which need to be documented on `INSTALL.rst`

Reply:
You are right!